packages/hardhat-core/src/internal/solidity/resolver.ts

```diff
@@ -287,14 +287,13 @@ export class Resolver {
       }
     }
 
-    let nodeModulesPath = path.dirname(path.dirname(packageJsonPath));
-    if (this._isScopedPackage(sourceName)) {
-      nodeModulesPath = path.dirname(nodeModulesPath);
-    }
+    let packageRoot = path.default.dirname(packageJsonPath);
```

Comment:
Is this a Yarn 2+ specific API? I can't find any docs about `path.default`.

Reply:
I corrected it to `path.dirname`.